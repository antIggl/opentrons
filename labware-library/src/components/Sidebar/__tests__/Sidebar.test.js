// @flow
// Sidebar component tests
import * as React from 'react'
import { shallow } from 'enzyme'

import { Sidebar } from '..'

jest.mock('../../../definitions')

describe('Sidebar', () => {
  const filters = { category: 'all', manufacturer: 'all' }

  test('component renders', () => {
    const tree = shallow(<Sidebar filters={filters} />)

    expect(tree).toMatchSnapshot()
  })

  test('renders a <nav>', () => {
    const tree = shallow(<Sidebar filters={filters} />)

    expect(tree.find('nav')).toHaveLength(1)
  })
})
