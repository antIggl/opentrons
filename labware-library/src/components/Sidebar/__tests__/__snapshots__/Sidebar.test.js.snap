// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component renders 1`] = `
<nav
  className="sidebar"
>
  <LabwareGuide />
  <withRouter(FilterManufacturerComponent)
    filters={
      Object {
        "category": "all",
        "manufacturer": "all",
      }
    }
  />
  <FilterCategory
    filters={
      Object {
        "category": "all",
        "manufacturer": "all",
      }
    }
  />
  <FilterReset
    filters={
      Object {
        "category": "all",
        "manufacturer": "all",
      }
    }
  />
</nav>
`;
